virtualizarr/tests/test_writers/test_icechunk.py

```diff
@@ -528,6 +528,7 @@ def test_append_virtual_ref_with_encoding(
             xrt.assert_equal(new_ds, expected_ds)
 
     ## When appending to a virtual ref with encoding, it succeeds
+    @pytest.mark.xfail
```

Comment:
Can you add a reason for this xfail marker?

Reply:
Apparently there was no reason for it - removed in https://github.com/zarr-developers/VirtualiZarr/pull/477/commits/6bf2615c1d80b30a392c1a5c9d742c3a69f6d6f5